src/main/resources/db/migration/V2024_03_01_1222__CIV-12657-claim-issue-hwf-scenario.sql

```diff
@@ -0,0 +1,50 @@
+/**
+ * Add scenario
+ */
+INSERT INTO dbs.scenario (name, notifications_to_delete, notifications_to_create)
+VALUES ('Scenario.AAA7.ClaimIssue.HWF.Requested', '{"Notice.AAA7.ClaimIssue.ClaimSubmit.Required"}',
+        '{"Notice.AAA7.ClaimIssue.HWF.Requested" : []}');
+
+/**
+ * Add notification template
+ */
+INSERT INTO dbs.dashboard_notifications_templates (template_name, title_En, title_Cy, description_En, description_Cy
+                                                  ,notification_role)
+VALUES ('Notice.AAA7.ClaimIssue.HWF.Requested', 'We’re reviewing your help with fees application' , 'We’re reviewing your help with fees application',
+        'You’ve applied for help with the claim fee. You’ll receive an update in 5 to 10 working days.',
+        'You’ve applied for help with the claim fee. You’ll receive an update in 5 to 10 working days.',
+        'CLAIMANT');
+
+/**
+ * Add task list items
+ */
+INSERT INTO dbs.task_item_template (task_name_en, category_en, task_name_cy, category_cy, template_name,
+                                    scenario_name, task_status_sequence, role,task_order)
+values ('<a href={VIEW_CLAIM_URL}  rel="noopener noreferrer" class="govuk-link">View the claim</a>', 'The claim',
+        '<a href={VIEW_CLAIM_URL}  rel="noopener noreferrer" class="govuk-link">View the claim</a>',
+        'The claim', 'Claim.View', 'Scenario.AAA7.ClaimIssue.HWF.Requested', '{3, 3}', 'CLAIMANT', 1),
+       ('<a href={VIEW_INFO_ABOUT_CLAIMANT_URL}  rel="noopener noreferrer" class="govuk-link">View information about the claimant</a>', 'The claim',
+        '<a href={VIEW_INFO_ABOUT_CLAIMANT_URL}  rel="noopener noreferrer" class="govuk-link">View information about the claimant</a>',
+        'The claim', 'Claim.Claimant.Info', 'Scenario.AAA7.ClaimIssue.HWF.Requested', '{3, 3}', 'CLAIMANT', 2),
+       ('<a>View the response to the claim</a>', 'The response','<a>View the response to the claim</a>',
+        'The response', 'Response.View', 'Scenario.AAA7.ClaimIssue.HWF.Requested', '{1, 1}', 'CLAIMANT', 3),
+       ('<a href={VIEW_INFO_ABOUT_DEFENDANT_URL}  rel="noopener noreferrer" class="govuk-link">View information about the defendant</a>', 'The response',
```

Comment:
I think the placeholder may need to be `VIEW_INFO_ABOUT_DEFENDANT` to match CUI dashboardInterpolationService.ts `setDashboardValues()` function.

Reply:
This change is from another PR https://github.com/hmcts/civil-service/pull/4136